modules/rgbd/src/large_kinfu.cpp

```diff
@@ -136,6 +143,10 @@ class LargeKinfuImpl : public LargeKinfu
 
     int frameCounter;
     Affine3f pose;
+
+    // Loop Closure
+    bool useLCD;
```

Comment:
Do we need `useLCD` field? We can check `lcd` pointer instead

Reply:
Thanks for code reviewing. it will be updated in the next version.